tests/engine/forced_checks.robot

```diff
@@ -13,7 +13,7 @@ Library	../resources/Engine.py
 
 *** Test Cases ***
 EFHC1
-	[Documentation]	Engine is configured with hosts and we force checks on one 5 times on bbdo2
+	[Documentation]	Engine is configured with hosts and we force checks on one host 5 times on bbdo2
```

Comment:
this is again an external command test on force host checks. We check that engine recognizes the force checks.